public/offline.html

```diff
@@ -0,0 +1,124 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
+
+    <base href="/" />
+
+    <link rel="apple-touch-icon" href="/logo192.png" />
+    <link rel="manifest" href="/manifest.json" />
+    <link
+      href="/static/fonts/montserrat.css"
+      type="text/css"
+      rel="stylesheet"
+    />
+
+    <title>Zeeguu</title>
+  </head>
+  <style>
+    .contentDiv {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      font-family: "Monsterrat", sans-serif;
+    }
+
+    .capitalizeBold {
+      text-transform: capitalize;
+      font-weight: bold;
+    }
+  </style>
+
+  <body>
+    <div id="offlineRoot" class="contentDiv">
+      <img src="/logo192.png" />
+      <h1>You seem to be offline</h1>
+      <h2>Please return to Zeeguu when you are back online!</h2>
+      <div id="wordsList">Loading cached words...</div>
+    </div>
+
+    <script>
+      async function displayCachedWords() {
+        try {
+          const sessionId = document.cookie.split(";")[0].split("=")[1];
```

Comment:
can you fix this pls? this is fixed in index.html but still hardcoded here.

Reply:
Oh yeah, I see. I changed it.